winch/codegen/src/codegen/context.rs

```diff
@@ -320,6 +320,21 @@ impl<'a> CodeGenContext<'a> {
         }
     }
 
+    /// Assigns the value at the stack top to the local at the given index
```

Comment:
This comment is a little ambiguous, especially in combination with the context of `local.tee` -- I would expect it to use the value at the TOS, but not pop it. Could we say "Pops the value at the stack top and assigns it to the local ..."?

Reply:
Sure, that makes sense to me. Done, thanks!